### PR TITLE
Remove header's links in mobile view

### PR DESCRIPTION
**Pull request by svillafe**
**[DO NOT MERGE]** Remove header's links in mobile view. This is a test feature, do not merge it.

### CODE REVIEW

**Comment by svillafe** [comment]
beforeafter

File: frontend/assets/stylesheets/utilities/_util-visibility.scss [hunk 1]
```diff
@@ -71,6 +72,12 @@
     }
 }
 
+.desktop-only {
+    @include mq($until: tablet) {
```
Comment (rtyley) [inline frontend/assets/stylesheets/utilities/_util-visibility.scss]: I'm not knowledgeable about scss, so I'm a bit confused here - you're creating a new style called `desktop-only`, but it says `$until: tablet` - can you explain what's going on here? How is it `desktop-only`?

File: frontend/assets/stylesheets/components/_navigation.scss [hunk 2]
```diff
@@ -129,7 +129,7 @@
         .global-navigation__scroll,
         .global-navigation__list,
         .global-navigation__item {
-            display: block;
+            display: block !important;
```
Comment (rtyley) [inline frontend/assets/stylesheets/components/_navigation.scss]: I realise that `!important` is being used elsewhere, but it's a fairly dangerous bit of CSS - it would be good to reduce it's usage if possible, are you sure it's needed here? Ideally, we would use BEM principles to avoid overwriting styles like this.

@SiAdcock on the frontend team (sits immediately behind us!) would be happy to discuss alternative ways of fixing this!

File: frontend/assets/stylesheets/utilities/_util-visibility.scss [hunk 3]
```diff
@@ -62,6 +62,7 @@
     }
 }
 
+
```
Comment (rtyley) [inline frontend/assets/stylesheets/utilities/_util-visibility.scss]: Minor nit - try to keep pull requests smaller by leaving out unnecessary changes - makes it easier to focus on the important changes.

**Comment by svillafe** [comment]
@SiAdcock please check this changes.

**Comment by SiAdcock** [comment]
Tidy work @svillafe  💎 💎 👍 💎 💎

**Comment by svillafe** [comment]
Please @rtyley review this change before merging it with master. Thank you.

**Comment by rtyley** [comment]
:+1:

**Comment by svillafe** [comment]
It is working in production.

picture 9
